### PR TITLE
miscellaneous

### PR DESCRIPTION
**Pull request by MiheerTamkhane**
Complete CL (P1). 
- saving dark mode in local storage.
- complete docs.
- clean up files.

### CODE REVIEW

File: components/modal/modal.css [hunk 1]
```diff
@@ -3,8 +3,8 @@
 .ct-modal {
   padding: 0 1rem;
   border: 1px solid var(--gray-500);
-  max-width: 350px;
-  min-width: 350px;
+  max-width: 320px;
+  min-width: 320px;
```
Comment (viru2001) [inline components/modal/modal.css:7]: you can use relative units here

File: styles/root.css [hunk 2]
```diff
@@ -1,9 +1,10 @@
+@import url("https://fonts.googleapis.com/icon?family=Material+Icons");
 :root {
   /* color theme of lib  */
-  --heading: #222831;
+  /* --heading: #222831;
   --text: #393e46;
-  --primary: #00adb5;
-  --body: #f9f9f9;
+  --primary: #02bfc9;
+  --body: #f9f9f9; */
```
Comment (viru2001) [inline styles/root.css]: if you will not use this code in future, then you can remove this

**Review by viru2001 — COMMENTED** [review]
I liked your UI and color palette , it is awesome. Also ,I suggested minor changes which you can do.

File: components/modal/modal.css (hunk 1 above)
Comment (MiheerTamkhane) [reply components/modal/modal.css:7]: Yup Thanks I keep forgetting that.

File: styles/root.css (hunk 2 above)
Comment (MiheerTamkhane) [reply styles/root.css]: oh let me see

**Review by bipin7yadav — APPROVED** [review]

**Review by someshtiwarii — APPROVED** [review]

**Pull request merged**